examples/wrapt_patching.py

```diff
@@ -1,27 +1,28 @@
 """Utilities for changing function signatures."""
```

Comment:
I currently get an error with this example

```python-traceback
(base) czirwc1macos2701:magicgui nsofroniew$ python examples/wrapt_patching.py
Traceback (most recent call last):
  File "examples/wrapt_patching.py", line 305, in <module>
    guis[func.__name__] = magicgui(func, **opts)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/function_gui.py", line 283, in magicgui
    return inner_func(function)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/function_gui.py", line 275, in inner_func
    app=app,
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/function_gui.py", line 74, in __init__
    sig = magic_signature(function, gui_options=param_options)
  File "/Users/nsofroniew/GitHub/magicgui/magicgui/signature.py", line 270, in magic_signature
    "keyword arguments (gui_options) MUST match parameters in the "
ValueError: keyword arguments (gui_options) MUST match parameters in the decorated function.
Got extra keys: {'ignore'}
```

Reply:
removing